src/main/java/seedu/masslinkers/logic/parser/AddCommandParser.java

```diff
@@ -37,8 +38,12 @@ public class AddCommandParser implements Parser<AddCommand> {
     public AddCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_NAME,
                 PREFIX_PHONE, PREFIX_EMAIL, PREFIX_TELEGRAM, PREFIX_GITHUB, PREFIX_INTEREST, PREFIX_MOD);
-        if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_TELEGRAM) || !argMultimap.getPreamble().isEmpty()) {
-            throw new ParseException(String.format(MESSAGE_INVALID_ARGUMENTS, AddCommand.MESSAGE_USAGE));
+        if (!argMultimap.getPreamble().isEmpty()) {
+            String invalidArgs = argMultimap.getPreamble();
+            throw new ParseException(String.format(MESSAGE_INVALID_ARGUMENTS, invalidArgs));
```

Comment:
Could add the AddCommand.MESSAGE_USAGE afterwards
Currently, doing `add ji xxsds`
gives `Invalid arguments 'ji xxsds' provided to the command!`

But we could add like what it should be after Invalid arguments...

Reply:
Fixed ✅